scalableminds/webknossos
# Persist mappingName and mappingIsLocked in NML

### URL of deployed dev instance (used for testing):
- https://mappingnml.webknossos.xyz

### Steps to test:
- Create annotation on dataset with agglomerate mapping, brush some (mapping will be locked)
- download + reupload
- Mapping of fallback layer should still be selected and locked
- Also test download+reupload with mapping but without volume brushing (mapping should be set but not locked)
- Also test without mapping (download+reupload should still run through without error)
- Beware of unrelated bug https://github.com/scalableminds/webknossos/issues/7823 – if your test dataset has a segment index file, you might have to remove that before you can test this PR

### Issues:
- fixes #7822

------
